src/main/java/com/rtb/projectmanagementtool/pages/HomePageServlet.java

```diff
@@ -0,0 +1,91 @@
+/** Servlet responsible getting data for loading home page */
+package com.rtb.projectmanagementtool.pages;
```

Comment:
We started off by packaging them by feature (user/project/task/auth, etc..), but now it's looking like we're packing them by layer.  If possible, I would suggest keeping your packaging convention consistent.